lib/rex_api_neues_rss.php

```diff
@@ -2,98 +2,67 @@
 
 class rex_api_neues_rss extends rex_api_function
 {
-    protected $published = true;  // Aufruf aus dem Frontend erlaubt
+    protected $published = true;  // Erlaubt den Aufruf aus dem Frontend
 
-    public function execute(): void
+    public function execute()
     {
         $domain_id = rex_request('domain_id', 'int', null);
         $lang_id = rex_request('lang_id', 'int', null);
         $category_id = rex_request('category_id', 'int', null);
 
-        if($category_id && $category = neues_category::get($category_id)) {
-            /** @var neues_category $category */
+        if ($category_id && $category = neues_category::get($category_id)) {
             $collection = neues_entry::findOnline($category_id);
-            $filename = 'rss.neues.' . rex_string::normalize($category->getName()) . '.xml';
+            $categoryname = rex_string::normalize($category->getName());
+            $filename = 'rss.neues.' . $categoryname . '.xml';
```

Comment:
Müsste hier nicht noch normalisiert werden fürs Dateisystem?

Reply:
ist doch schon in 15 geschehen

Reply:
obwohl , es wäre doch besser es etwas anders zu machen

Reply:
https://github.com/FriendsOfREDAXO/neues/commit/01e828e7f92374ec1fc923d0d382de8776837e99